src/main/java/org/shipkit/auto/version/NextVersionPicker.java

```diff
@@ -33,8 +33,17 @@ String pickNextVersion(Optional<Version> previousVersion, VersionConfig config,
         }
 
         if (!config.getRequestedVersion().isPresent()) {
-            String tag = runner.run("git", "describe", "--tags").trim();
+            String tag;
             String result;
+
+            try {
+                tag = runner.run("git", "describe", "--tags").trim();
+            } catch (Exception e) {
+                result = "0.0.1-SNAPSHOT";
+                explainVersion(log, result, "couldn't find tags in project");
```

Comment:
"couldn't find tags in project" -> "couldn't run 'git describe --tags' (were the tags fetched?)"